### ISSUE TITLE
Rule accounts_password_set_min_life_existing is not aligned with DISA STIG benchmark (GUI)

### ISSUE THREAD

**Issue by matusmarhefka** [ComplianceAsCode/content#9694]
#### Description of problem:
After remediating the GUI system with Bash remediation, the rule `accounts_password_set_min_life_existing` is reported as fail while its DISA's counterpart `SV-230364r627750_rule` is reported as pass.

#### SCAP Security Guide Version:
master as of HEAD bf8e854

#### Operating System Version:
RHEL8

#### Steps to Reproduce:

1. Perform clean RHEL8 installation (Server with GUI)
2. Harden the system with openscap using stig_gui profile (after the installation)
3. Evaluate stig_gui profile from this repo
4. Evaluate DISA benchmark

#### Actual Results:
`xccdf_org.ssgproject.content_rule_accounts_password_set_min_life_existing` fails
`xccdf_mil.disa.stig_rule_SV-230364r627750_rule` passes

#### Expected Results:
Both rules pass

**Comment by marcusburghardt** [comment]
This issue is probably fixed by #9955 . Let's observe the next round of tests.

**Comment by mildas** [comment]
Yes, it seems to be fixed. I don't see it failing in tests anymore. Closing
